docs/install.md

```diff
@@ -11,8 +11,8 @@ Beta is all about fixing these issues and improving Flarum. **Please don't use F
 Before you install Flarum, it's important to check that your server meets the requirements. To run Flarum, you will need:
 
 * **Apache** (with mod_rewrite enabled) or **Nginx**
-* **PHP 5.6+** with the following extensions: mbstring, pdo_mysql, openssl, json, gd, dom, fileinfo, tokenizer
-* **MySQL 5.5+**
+* **PHP 7.1+** with the following extensions: mbstring, pdo_mysql, openssl, json, gd, dom, fileinfo, tokenizer
```

Comment:
fileinfo should be removed, see https://github.com/flarum/core/commit/105dd093febbecdbd0964452511285bb8c2d0586.

Also, ordering them alphabetically will probably help in keeping this list up-to-date.